### PR TITLE
Created a python script to download the data.xlsx file from google dr…

### PR DESCRIPTION
**Pull request by ander324**
…ive using wget and then parses the .xlsx file into .csv files via pandas.

### CODE REVIEW

**Comment by ander324** [comment]
I like the bash script a lot more. It is very clean. You decide though =D

File: adhoc_reports/parseData.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import os
+import pandas as pd
```
Comment (Razamut) [inline adhoc_reports/parseData.py]: Whew, pandas is a pretty large library. Never a good idea to import the entire pandas just to use one or two methods. Import what you need.

File: adhoc_reports/parseData.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+from pprint import pprint
+
+os.system("wget --no-check-certificate \'https://spreadsheets.google.com/spreadsheet/ccc?key=1UHyMs5Havel4iEBMPVx7ZY4HMPuhRilIw59a5g733p8&output=xlsx\' -O data.xlsx")
```
Comment (Razamut) [inline adhoc_reports/parseData.py]: Exposing your credentials by explicitly hard coding it is a bad idea. The key=1UH**3p8 part should be hidden in a config file that should not be hosted on github; it should reside at the amazon instance node. Or at least passed as a command line argument.

File: adhoc_reports/parseData.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+wget --no-check-certificate 'https://docs.google.com/spreadsheets/d/1UHyMs5Havel4iEBMPVx7ZY4HMPuhRilIw59a5g733p8/export?gid=1795037732&format=csv' -O loans.csv
+wget --no-check-certificate 'https://docs.google.com/spreadsheets/d/1UHyMs5Havel4iEBMPVx7ZY4HMPuhRilIw59a5g733p8/export?gid=1851646659&format=csv' -O collections.csv
+wget --no-check-certificate 'https://docs.google.com/spreadsheets/d/1UHyMs5Havel4iEBMPVx7ZY4HMPuhRilIw59a5g733p8/export?gid=638160785&format=csv' -O person.csv
+wget --no-check-certificate 'https://docs.google.com/spreadsheets/d/1UHyMs5Havel4iEBMPVx7ZY4HMPuhRilIw59a5g733p8/export?gid=449217660&format=csv' -O connections.csv
```
Comment (Razamut) [inline adhoc_reports/parseData.sh]: Exposed credentials. depending on the settings in google, anyone with these urls can access those files.

**Review by Razamut — CHANGES_REQUESTED** [review]
based on the security issues, this code needs changes before hosting on here. And will not be merged in master for now until we conclude the problem cannot be solved using Haskell.

File: adhoc_reports/parseData.sh (hunk 3 above)
Comment (ander324) [reply adhoc_reports/parseData.sh]: I completely forgot about the public nature of this. Can you delete this branch? Then I can make a more publicly neutral version.

**Pull request closed without merging**